webapp/packages/core-blocks/src/Table/TableState.ts

```diff
@@ -9,30 +9,30 @@ import { action, computed, makeObservable, observable } from 'mobx';
 
 import { Executor, IExecutor } from '@cloudbeaver/core-executor';
 
-type Key = string | string[];
+type Key = any;
```

Comment:
please lets do proper types if possible. it would make debug easier if something about to happen here